api_mt/integrations/api_keys.md

```diff
@@ -0,0 +1,141 @@
+# How to obtain API keys for MT plugins  
+
+
+## Amazon Translate (AWS)  
+
+
+Create an **[AWS account](https://aws.amazon.com/)**
+![How to obtain an API key for Amazon Translate AWSpic1](trados/trados_for_amazon4.png)  
```

Comment:
The image has nothing to do with Trados though, right?

Reply:
I renamed and separated API-related images as @bittlingmayer noted